app/controllers/cart_controller.rb

```diff
@@ -21,6 +21,16 @@ def remove_item
     session[:cart].delete(params[:item_id])
     redirect_to '/cart'
   end
+  
+  def increment_decrement
```

Comment:
This method was really cool!  Glad we solved it as a team and that you got to give us the final rundown of how it worked!